controllers/userController.js

```diff
@@ -0,0 +1,11 @@
+const {User} = require('../models/models');
+
+class UserController {
+    async addUser(req, res) {
+        const {name, password} = req.body;
+        const user = await User.create({name, password});
```

Comment:
storing plain password in the DB is a bad approach. Consider turning it into hash and store hash instead of password